liblouis/lou_backTranslateString.c

```diff
@@ -980,12 +980,14 @@ makeCorrections(const TranslationTableHeader *table,
 		int *cursorPosition, int *cursorStatus, const TranslationTableRule **appliedRules,
 		int *appliedRulesCount, int maxAppliedRules) {
 	int pos;
+	int posIncremented;
 	int nextUpper = 0;
 	int allUpper = 0;
 	int allUpperPhrase = 0;
 	if (!table->corrections) return 1;
 	pos = 0;
 	output->length = 0;
+	posIncremented = 1;
```

Comment:
This is of course personal preference, but I'd combine line 983 and line 990 into 
``` C
int posIncremented = 1;
```